laalaa/apps/advisers/fixtures/initial_advisers.json

```diff
@@ -86184,7 +86184,8 @@
       2,
       3,
       7,
-      8
+      8,
+      14
```

Comment:
Why do we need these changes?

Reply:
These help setup the initial state. Not the main purpose but It's also useful for making sure there is data for end to end tests

Reply:
:+1: Gotcha.
Could we expose that intent in a separate commit (and message), if possible, please?

Reply:
🎉 Added second commit(with message) stating this

Reply:
Might be worth switching our fixtures to have the code as the natural key at some point (as opposed to using the pks), but not for now.